x-pack/plugins/siem/public/components/tables/helpers.test.tsx

```diff
@@ -25,6 +76,28 @@ describe('Table Helpers', () => {
       expect(toJson(wrapper)).toMatchSnapshot();
     });
 
+    test('it returns empty value when rowItems is undefined', () => {
+      const rowItems = getRowItemDraggables({
+        rowItems: undefined,
+        attrName: 'attrName',
+        idPrefix: 'idPrefix',
+        displayCount: 0,
+      });
+      const wrapper = shallow(<TestProviders>{rowItems}</TestProviders>);
+      expect(wrapper.html()).toBe(getEmptyValue());
+    });
+
+    test('it returns empty value when rowItems is null', () => {
```

Comment:
++ on pushing the `undefined`/`null`-check up to `getRowItemDraggables()`! This really cleans up the column declarations! :)